packages/app/src/cli/utilities/extensions/configuration.ts

```diff
@@ -106,6 +106,8 @@ export function getUIExtensionSurface(uiExtensionType: UIExtensionTypes) {
   switch (uiExtensionType) {
     case 'checkout_ui_extension':
       return 'checkout'
+    case 'ui_extension':
+      return 'checkout_beta'
```

Comment:
Why does this get mapped to `checkout_beta`?

Reply:
Yeah, this is wrong.  I've updated this to `checkout`

In a follow on PR we'll have to reconsider this entirely since the new extension has multiple surfaces:

This affects:

1. Constructing the JSON store which is returned by the extensions dev server.
2. Constructing the redirect URL for an extension.
3. The extensionConfig we pass to go (But since this is going 💥 we don't need to worry about this).

It looks like all these points are handled in pre-exisitng tickets, so I'll resolve this.  Please re-open if I've missed anything.